test-integ/topoutil/fixtures.go

```diff
@@ -36,19 +38,61 @@ func NewFortioServiceWithDefaults(
 			"server",
 			"-http-port", strconv.Itoa(httpPort),
 			"-grpc-port", strconv.Itoa(grpcPort),
+			"-tcp-port", strconv.Itoa(tcpPort),
 			"-redirect-port", "-disabled",
 		},
 	}
 
 	if nodeVersion == topology.NodeVersionV2 {
 		svc.Ports = map[string]*topology.Port{
-			// TODO(rb/v2): once L7 works in v2 switch these back
-			"http":     {Number: httpPort, Protocol: "tcp"},
-			"http-alt": {Number: httpPort, Protocol: "tcp"},
-			"grpc":     {Number: grpcPort, Protocol: "tcp"},
-			// "http":     {Number: httpPort, Protocol: "http"},
-			// "http-alt": {Number: httpPort, Protocol: "http"},
-			// "grpc":     {Number: grpcPort, Protocol: "grpc"},
+			"http":  {Number: httpPort, Protocol: "http"},
```

Comment:
Though I am not using fortio here, this is an in-passing update b/c the underlying plumbing was fixed in 1.17.0